utils/cautils/acmeutils.go

```diff
@@ -361,32 +360,45 @@ func newACMEFlow(ctx *cli.Context, ops ...acmeFlowOp) (*acmeFlow, error) {
 }
 
 func (af *acmeFlow) GetCertificate() ([]*x509.Certificate, error) {
+	var (
+		orderPayload []byte
+		clientOps    []ca.ClientOption
+		idents       []acme.Identifier
+		nor          interface{}
+	)
+
+	// Public CA does not support NotBefore and NotAfter attributes in orders.
```

Comment:
I'm not sure we want to error here. Let's Encrypt is not the only public ACME CA and don't think there's anything in the spec that would prevent `not-after` and `not-before` being sent. Instead, we probably want to make these values pointers, so that they can be set to nil if the values are empty - rather than the zero time value.

Reply:
Make sense.